frontend/yarn.lock

```diff
@@ -0,0 +1,10021 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Apparently you real are supposed to commit these 10K lines:
https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

Reply:
yeah. that's how you get reproducible builds. it's also handy when you add/update a dependency to see the diff to this file to see everything else that changes. I'm fine with this being checked in (but not that it is 10,000 lines. i mean really? how many leftpad's do we need?)